cypress/support/step_definitions/cohort-save.js

```diff
@@ -100,10 +100,53 @@ when('I save the Cohort with name {string}', (cohortName) => {
   cy.contains('Save cohort').click();
 });
 
+when('I create a cohort with {string} dimension constraint', (dimension) => {
+  cy.toggleNode('Public Studies ');
+  cy.drag('CSR').drop(0);
+  cy.drag('EHR').drop(0);
+  cy.contains('i', 'and').click();
+  cy.contains('i', 'or').should('be.visible');
+
+  cy.get('.gb-constraint-cohort-type-dropdown').get('.ui-dropdown').click();
+  cy.get('.ui-dropdown').contains(dimension).click();
+  cy.get('.update-btn').eq(0).click();
+});
+
 then('the cohort {string} is saved', (cohortName) => {
   cy.get('.ng-trigger-tabContent').eq(1).contains(cohortName);
 });
 
 then('the cohort {string} is deleted', (cohortName) => {
   cy.get('.ng-trigger-tabContent').eq(1).contains(cohortName).should('not.be.visible');
 });
+
+then('the current cohort has biomaterial selected', () => {
+  cy.get('.gb-constraint-cohort-type-dropdown').eq(0).contains('Biomaterial ID');
+  cy.contains('CSR');
+  cy.contains('EHR');
+  cy.get('.gb-constraint-cohort-type-dropdown').should('have.length', 1); //FIXME <- should be 1, is 2 - additional subselection
+});
+
+when('I create a cohort with multiple dimensions constraint', () => {
+  cy.toggleNode('Public Studies ');
+  cy.toggleNode('CSR');
+  cy.toggleNode('01. Patient information');
+  cy.drag('02. Gender').drop(0);
+
+  cy.get('label').contains('f (5), m (4)').click();
+  cy.removeChip('m (4)');
+
+  cy.get('.gb-constraint-cohort-type-dropdown').get('.ui-dropdown').eq(0).click();
+  cy.get('.gb-constraint-cohort-type-dropdown').eq(0).contains('Biosource ID').click();
+
+  cy.drag('CLINICAL_TRIAL ').drop(1);
+  cy.get('.update-btn').eq(0).click();
+
+})
+
+then('the current cohort has multiple dimensions selected', () => {
+  cy.get('.gb-constraint-cohort-type-dropdown').eq(0).contains('Biomaterial ID');
+  cy.get('.gb-constraint-cohort-type-dropdown').eq(1).contains('patient');
+  cy.get('.gb-constraint-cohort-type-dropdown').should('have.length', 2); //FIXME
```

Comment:
'FIXME': is it fixed? :D

Reply:
not yet, I mentioned it in the description of this PR ("2 issues left") - so this is caused by the second one

Reply:
now it is fixed